src/plugins/conditionals/conditionals.c

```diff
@@ -785,6 +787,129 @@ static CondResult evaluateKey (const Key * meta, const Key * suffixList, Key * p
 	return TRUE;
 }
 
+static KeySet * getDependencies (const char * expr, Key * cur, Key * parentKey)
+{
+	char * localExpr = strdup (expr);
+	char * ptr = localExpr;
+	KeySet * deps = ksNew (0, KS_END);
+	int isEscaped = 0;
+	int hasQuote = 0;
+	for (; *ptr; ++ptr)
```

Comment:
Shouldn't this parsing code be part of libmeta (later)? Some generic link detection would be nice.